css/style.css

```
@@ -1 +1,18 @@
-/* No CSS *//*# sourceMappingURL=style.css.map */
\ No newline at end of file
+/* No CSS *//*# sourceMappingURL=style.css.map */
+
+.card-adolfo {
+    border: 1px solid rgb(128, 68, 0);
+    width: 205px;
+  }
+  
+ img {
+    width: 200px;
+    margin: 0 auto;
+  }
+  
+  
+
+
+  .container-adolfo {
+    padding: 2px 1px;
+  }
```

Trata de anidar tus selectores...

```suggestion
.card-adolfo {
    border: 1px solid rgb(128, 68, 0);
    width: 205px;
    img {
    width: 200px;
    margin: 0 auto;
  }
  }
  
 
  

```